cmd/gotypeconv/main.go

```diff
@@ -21,6 +21,12 @@ import (
 	"golang.org/x/tools/go/loader"
 )
 
+var unused = 1
+
+var NoComment = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var NoComment should have comment or be unexported